webview/src/stories/Table.stories.tsx

```diff
@@ -61,10 +62,7 @@ const tableData = getTableState({
   selectedBranches: [],
   selectedForPlotsCount: 2,
   showOnlyChanged: false,
-  sorts: [
```

Comment:
Sorts used to be a default addition to the storybooks but since `tableData` rows are different when they are sorted, I removed the default sorts array (sorts are still shown in our new sorted fixture and in the `deeplyNested` fixture).